proto/frequenz/api/common/v1/metrics/metric_sample.proto

```diff
@@ -46,19 +46,19 @@ message AggregatedMetricSample {
 }
 
 // MetricSampleVariant serves as a union type that can encapsulate either a
-// `SimpleMetricSample` or an `AggregatedMetricSample`.
+// `SimpleMetricValue` or an `AggregatedMetricValue`.
 //
 // This message is designed to offer flexibility in capturing different
 // granularities of metric samples—either a simple single-point measurement
 // or an aggregated set of measurements for a metric.
 //
-// A `MetricSampleVariant` can hold either a `SimpleMetricSample` or an
-// `AggregatedMetricSample`, but not both simultaneously. Setting one will
+// A `MetricSampleVariant` can hold either a `SimpleMetricValue` or an
+// `AggregatedMetricValue`, but not both simultaneously. Setting one will
 // nullify the other.
 message MetricSampleVariant {
```

Comment:
This needs to be renamed to `MetricValueVariant` and `metric_sample_type` to `metric_value_type`

Reply:
Good catch! :bow: 

Fixed